### PR TITLE
style: change quotes to double quote

### PR DESCRIPTION
**Pull request by Muunatic**
change single quotes to double quote writing style

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// import fetch, { Response } from "node-fetch";
```
Comment (HarunamiYaki) [inline src/index.ts:1]: This shouldn't have been there
```suggestion
```

**Review by HarunamiYaki — DISMISSED** [review]
I noticed that a line has been removed in [`feat(API)!: Replace fetch with Node.js 18 module` #38](https://github.com/Typeslint/github-AutoResponse/pull/38) which should not have been. Could you please remove this line from your changes?

File: src/index.ts (hunk 1 above)
Comment (Muunatic) [reply src/index.ts]: fixed, thanks!

**Review by HarunamiYaki — APPROVED** [review]

**Comment by Muunatic** [comment]
ready to merge

**Pull request merged**